app/exports/qualifications_export.yml

```diff
@@ -11,108 +11,108 @@ common_columns:
 custom_columns:
   gcse_maths_grade:
     type: string
-    description: GCSE maths grade
+    description: GCSE maths grade. [See list of GCSE grades](/support/docs/qualifications/#gcses)
```

Comment:
This works locally, and the internet seems to think that this will work across the different environments. But I wanted to call it out for a quick sanity check.